Makefile

```diff
@@ -73,6 +73,7 @@ list_merge_keys: ## List all GPG keys allowed to sign merge commits.
 .PHONY: globals
 PASSWORD_STORE_DIR?=${HOME}/.paas-pass
 globals:
+	$(eval export ENABLE_MORNING_DEPLOYMENT=false)
```

Comment:
I don't think we should set this when https://github.com/alphagov/paas-cf/pull/962/files#diff-ae4438819742e1613315377720ffa024R153 has a default of `false` because it would follow the pattern of other feature flags and prevent the two settings getting out-of-sync.